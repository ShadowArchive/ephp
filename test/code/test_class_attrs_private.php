<?php

class A {
    private $a = 100;
    public function sum($b) {
        return $a + $b;
    }
}


class B extends A {
}

$b = new B;
print $b->sum(10) . "\n";

