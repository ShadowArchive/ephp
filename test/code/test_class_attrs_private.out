
Notice: Undefined variable: a in {{CWD}}/test/code/test_class_attrs_private.php on line 6
10
